cmd/merlinagent/main.go

```diff
@@ -45,7 +47,7 @@ var debug = false
 var verbose = false
 var mRun = true
 var hostUUID = uuid.NewV4()
-var url = "https://127.0.0.1:443/"
+var url = "https://127.0.0.1:4433/"
```

Comment:
Default port should stay at `443`.

Reply:
Changing it back, I like to avoid having to run as root.

Reply:
If it helps, you can specify the URL when running it from the command line with `merlinServer.exe -url https://127.0.0.1:4433/`